webapp/channels/src/packages/mattermost-redux/src/reducers/entities/posts.ts

```diff
@@ -1623,6 +1637,9 @@ export default function reducer(state: Partial<PostsState> = {}, action: Generic
         // whether this particular view has messages that are hidden
         // because of the cloud workspace limit.
         limitedViews: limitedViews(state.limitedViews, action),
+
+        // Object mapping state keys to booleans indicating errors are present
+        errors: fetchErrors(state.errors, action),
```

Comment:
@M-ZubairAhmed @hmhealey will defer to you folks, for changes in redux store

Reply:
Yeah, this is something we probably want to do differently. Generally, we'd want the component that uses a piece of data (in this case, the `PostEditHistory`) to be responsible for fetching the data so that it can handle the error itself without us having to store it in Redux. We previously had tried to store more errors there, but that didn't end up working out in most cases.

For this, I see that the `EditPost` and `PostEditedIndicator` are responsible for loading the post edit history, but what if instead of that, we had them just open the `PostEditHistory` RHS if that's needed, and then the `PostEditHistory` can fetch the data it needs in a `useEffect`? Then, if that request fails, it can set some internal state to hold the error. That way, we don't need to store anything extra in the redux store

Reply:
I should have some more time in the coming weeks to take a stab at these adjustments. Once I have something working I'll revive this PR.